src/Rules/DisallowSetupAndConstructorRule.php

```diff
@@ -40,7 +40,7 @@ public function processNode(Node $node, Scope $scope): array
         $node = $node;
 
         $classReflection = $scope->getClassReflection();
-        if ($classReflection === null || ! UnitTestRuleHelper::isUnitTest((string) $scope->getNamespace(), $classReflection->getName(), $this->unitTestNamespaceContainsString)) {
+        if ($classReflection === null || ! UnitTestRuleHelper::isUnitTest((string) $scope->getNamespace(), $classReflection->getName(), $this->unitTestNamespace)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class 'ikvasnica\PHPStan\Rules\UnitTestRuleHelper' in method 'processNode'.](https://app.codacy.com/manual/ikvasnica/phpstan-clean-test/pullRequest?prid=4930471)